reactioncommerce/reaction
# Fixed other admin users not being able to add/edit/remove product tag

Resolves #2426 ,

New users with some administrative permissions couldn't access `TagList` component on the PDP because an `editable` prop that checks whether a user is a customer was not passed to the component . So I added the missing prop ie.`editable`' in the `TagList` component being used in product settings view.

#### How to test
- Sign in as admin.
- Invite a new shop manager or admin.
- Open a private tab in your browser and follow the link in the invitation mail
- Once you are signed in , open a product and view it's PDP panel.
- Notice that on the Tags Card now you can add/edit/remove a tag.
